bindings/pyroot/cppyy/CPyCppyy/src/CPPExcInstance.cxx

```diff
@@ -150,6 +150,77 @@ static PyObject* ep_richcompare(CPPExcInstance* self, PyObject* other, int op)
     return PyObject_RichCompare(self->fCppInstance, other, op);
 }
 
+//----------------------------------------------------------------------------
+static int ep_nonzero(CPPExcInstance* self)
+{
+   if (self->fCppInstance)
+      return PyObject_IsTrue(self->fCppInstance);
+
+   return 0;
+}
+
+//-----------------------------------------------------------------------------
+static PyNumberMethods ep_as_number = {
```

Comment:
What about the other methods? If we treat the `CPPExcInstance_Type` as number and follow the Python number protocol, we'll still run into undefined operations, like adding stuff. Do we error out gracefully with a `0` in the type field or do we crash with a segfault?

Reply:
Perfect!